src/savers/meson.build

```diff
@@ -0,0 +1,8 @@
+subsaver_dep = []
+
```

Comment:
See how loaders do this:

if get_option('loaders').contains('tvg') == true
    subdir('tvg')
    message('Enable TVG Loader')
endif